src/main/resources/assets/opencomputers/loot/mineos/installer.lua

```diff
@@ -0,0 +1,385 @@
+local component = require("component")
+local computer = require("computer")
+local term = require("term")
+local unicode = require("unicode")
+local event = require("event")
+local fs = require("filesystem")
+local seri = require("serialization")
+local shell = require("shell")
+local gpu = component.gpu
+
+-----------------Проверка компа на соответствие сис. требованиям--------------------------
+
+shell.execute("cd ..")
+shell.setWorkingDirectory("")
+
+--Создаем массив говна
+local govno = {}
```

Comment:
This is like an english word "shit". We're creating a table of shit - and if installer faces some shitty RAM modules, video cards, etc - table will fill with information about this shit. And, of course, installer will close and show this information to user.

I hope such a minor thing like name of the variables will not create additional problems.